### PR TITLE
Feature/upgrade standard token contract

### PR DESCRIPTION
**Pull request by surabhil**
- Use EOSIO_DISPATCH instead of EOS_ABI  
- Update contract to use symbol_code instead of symbol_name
- Use eosio replacement of account_name to name 
-  Explicitely use name.value to query in table as it converts name type to uint64_t
- Update multi indexes to the new format as per eosio
- Add class type (eosio::contract("ore.standard_token")) in the class definition 
- Anotate all tables and actions with eosio::action/table for abi generation
-  Add open and close actions as added in eosio.token contract in EOSIO/eoiso.contracts latest version
-  Use TABLE and ACTION macros

### CODE REVIEW

**Review by blackwatertepes — APPROVED** [review]
LGTM, although, I'd wait for another Review, before merging.

**Comment by bas4r** [comment]
Looks good to me as well

**Pull request merged**
